src/components/FeedbackForm/index.js

```diff
@@ -51,12 +52,21 @@ export const FeedbackForm = () => {
             if (res.status === 201) {
               setInputVal({ value: '', errorMsg: '' });
               setDropDownValue({ ...dropDownValue, value: '' });
+              setDoneMsg({ value: 'Msg is Sended', status: true });
+              setInterval(() => {
+                setDoneMsg({ value: '', status: false });
+              }, 3000);
```

Comment:
it doesn't handle like that!
there remove if condition and add `.then()` for success like you did `.catch()` for errors